install/Knative-with-Minishift.md

```diff
@@ -147,51 +166,65 @@ bash <(curl -s https://raw.githubusercontent.com/knative/docs/master/install/scr
 1. Install Knative serving:
 
     ```shell
-    curl -L https://storage.googleapis.com/knative-releases/serving/latest/release-lite.yaml \
-    | sed 's/LoadBalancer/NodePort/' \
-    | oc apply -f -
+     oc apply -f https://storage.googleapis.com/knative-releases/serving/latest/release-no-mon.yaml
```

Comment:
I wonder if the doc should use `latest` or the released `0.1.1` version  

https://github.com/knative/serving/releases/download/v0.1.1/release-no-mon.yaml

Reply:
@matzew  lets use latest until we have one good dot release that is stable